src/v2/Apps/Artwork/Components/ArtworkSidebar/__tests__/ArtworkSidebar.jest.tsx

```diff
@@ -48,7 +48,6 @@ describe("ArtworkSidebar", () => {
       featureFlags: {
         "artwork-page-create-alert": {
           flagEnabled: false,
-          variant: { name: "enabled", enabled: false },
```

Comment:
As unleash mentions in their [docs](https://docs.getunleash.io/advanced/toggle_variants#the-disabled-variant)

> This is a fallback variant that Unleash uses to represent the lack of a variant.